src/frontend/src/flows/dappsExplorer/dapps.ts

```diff
@@ -0,0 +1,19 @@
+// The list of dapps. This is derived from https://github.com/dfinity/portal:
+// * Only dapps using II are used
+// * All relevant logos are copied to II's assets
+// * Some logos are converted to webp
+import dappsJson from "./dapps.json";
```

Comment:
would it be interesting to lazy load the json file?

Reply:
Makes sense, will do separately (code was only moved)

Reply:
done: https://github.com/dfinity/internet-identity/pull/1441